template/node/Cargo.toml

```diff
@@ -77,14 +82,8 @@ substrate-build-script-utils = { version = "3.0.0", git = "https://github.com/pa
 
 [features]
 default = ["aura", "with-rocksdb-weights"]
-aura = [
-	"sc-consensus-aura",
-	"frontier-template-runtime/aura",
-]
-manual-seal = [
-	"sc-consensus-manual-seal",
-	"frontier-template-runtime/manual-seal",
-]
+aura = ["frontier-template-runtime/aura"]
```

Comment:
the current implementation only has the sealing cli option to judge whether to enable the manual seal mode, the default is aura + grandpa.
I think this can reduce the duplication of code caused by different features in the service

Reply:
We could even get rid of the `aura` and `manual-seal` features if possible, but this would require adding a value in genesis to be checked in the `OnTimestampSet` part.

Reply:
like:

```rust
pub struct ConsensusOnTimestampSet<T>(PhantomData<T>);
impl<T: pallet_aura::Config> OnTimestampSet<T::Moment> for ConsensusOnTimestampSet<T> {
    fn on_timestamp_set(moment: T::Moment) {
        if storage::EnableManualSeal::get() {
            return
        }
        <pallet_aura::Pallet<T> as OnTimestampSet<T::Moment>>::on_timestamp_set(moment)
    }
}
```